mkdocs/faq.md

```diff
@@ -47,7 +47,7 @@ You can delete the `long_term_memory` folder and restart the Cat!
 
 Please check if you have a valid credit card connected or if you have used up all the credits of your OpenAI trial period.
 
-#### Everything works in localhost but not on another server
+#### Everything works in localhost but not on another server or needs an auth_key never configured
```

Comment:
what do you think putting this in a separate question? e.g. something like "why the admin panel is asking for an auth key?"

Reply:
i like it!